nuxt.config.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
    ** Axios module configuration
    */
   axios: {
-    baseURL: process.env.AXIOS_BASE_URL,
+    baseURL: process.env.NODE_ENV == "production" ? '' : process.env.AXIOS_BASE_URL,
```

Comment:
Why is baseURL not needed in prod?

Reply:
setting baseURL to empty string cause the axios to use default which is the current endpoint.

But still no good. this branch is not working yet.